.buildkite/pipeline.yml

```diff
@@ -138,18 +138,20 @@ steps:
   - label: ":jenkins: Release - Package Registry Distribution"
     key: "release-package-registry"
     trigger: "package-registry-release-package-registry-distribution"
+    async: true
     build:
       branch: "main"
       meta_data:
         DOCKER_TAG: "${BUILDKITE_TAG}"
     if: "build.env('BUILDKITE_TAG') != ''"
     depends_on:
       - step: "release-test"
-        allow_failure: false
+        allow_failure: true
 
   - trigger: "fleet-server-package-mbp"
     label: ":esbuild: Downstream - Package"
     key: "downstream-package"
+    async: true
```

Comment:
In Jenkins, this trigger step was asynchronous
https://github.com/elastic/fleet-server/blob/396d745bc1081190ff5912dcf0b05a266fe56f6a/.ci/Jenkinsfile#L228